pkg/server/proto/server.proto

```diff
@@ -1371,6 +1411,17 @@ message Ref {
       string name = 2;
     }
   }
+
+  message AddOnDefinition {
+    oneof ref {
```

Comment:
Maybe there's a better name for this other than `ref`? Otherwise, I think the proto will look like `Ref.Ref.<name|id>` which is a little odd, but also we have a lot of odd protobuf nested structures so maybe not the weirdest instance of this

Reply:
I renamed it to `identifier`!